### DISCUSSION TITLE
Sub Forms

### DISCUSSION THREAD

**Discussion by hardc0l2e** [Budibase/budibase#2207] [General]
Is it possible to have a sub form (CRUD) while inside the main table?

Example:
Main table = visitor information
Second table = visitor schedules

I tried to make relationship as "one Main Table, many Second Table", I just know how to properly setup their form?

The process is like this one:
1. Create/Edit the visitor information
2. Add multiple schedules (dates)

![20210801_11 54 19 PM-HSC_Portal_-_Google_Chrome](https://user-images.githubusercontent.com/1472421/127777337-6dfd5f23-61c6-416c-81e1-ac7583024ffd.png)

**Comment by mjashanks** [comment]
Hey @hardc0l2e - does this solve it for you: https://github.com/Budibase/budibase/discussions/2043 ?

**Reply by hardc0l2e** [reply]
Sorry for the duplicate posting, I found that discussion later and working now, thanks.
